s3/src/main/scala/akka/stream/alpakka/s3/javadsl/S3Client.scala

```diff
@@ -421,49 +423,49 @@ final class S3Client(s3Settings: S3Settings, system: ActorSystem, mat: Materiali
               MetaHeaders(Map()),
               sse)
 
+  private def toJava[M](
+      download: (akka.stream.scaladsl.Source[ByteString, M], Future[scaladsl.ObjectMetadata])
+  ): JPair[Source[ByteString, M], CompletionStage[ObjectMetadata]] = {
+    val (stream, meta) = download
+    JPair(stream.asJava, meta.map(metaDataToJava)(mat.executionContext).toJava)
+  }
+
   /**
    * Downloads a S3 Object
    *
    * @param bucket the s3 bucket name
    * @param key the s3 object key
-   * @return A [[Source]] of [[ByteString]] that materializes into a [[CompletionStage]] containing the [[ObjectMetadata]]
+   * @return A [[Pair]] with a [[Source]] of [[ByteString]], and a [[CompletionStage]] containing the [[ObjectMetadata]]
```

Comment:
Links to other classes almost always need to be fully qualified in Scaladoc. (But it's wront in many places in Alpakka.)

Reply:
added fully qualified `Pair`, let me know if you want to update `Source`, `CompletionStage`, etc as well.

Reply:
Yes, please.